assemblyline_core/metrics/es_metrics.py

```diff
@@ -523,6 +523,7 @@ def try_run(self):
                        ['es_cluster', 'es_nodes', 'es_indices'])
 
         while self.running:
+            self.heartbeat()
```

Comment:
Again there is a time.sleep at the end of this loop. This might be longer that the 30 sec health check timeout

Reply:
I think its fine in this case, the loop is hard coded to try to run every 10 seconds, with taking more than 5 seconds to finish its work raising warnings. By default the health check has to fail three times in a row, so it would need to take more than 130 seconds before we saw the container marked unhealthy.